cloudformation/calculation-athena.yaml

```diff
@@ -201,6 +201,37 @@ Resources:
 
         Workgroup: !Ref AthenaTransactionWorkgroup
 
+  BillingAndTransactionsCuratedView:
+    Type: Custom::AthenaView
+    DependsOn: TransactionsCuratedView
+    Properties:
+      ServiceToken: !GetAtt CustomAthenaViewResourceFunction.Arn
+      View:
+        Database: !Ref CalculationsDB
+        Name: btm_billing_and_transactions_curated
+        Query: |
+          SELECT 
+              b.vendor_name,
+              b.service_name,
+              COALESCE(b.year,t.year)                                                                                                         AS year,
+              COALESCE(b.month,t.month)                                                                                                       AS month,
```

Comment:
These fields can actually be blank in one table or the other, and it will still include a row for them, I believe thanks to the FULL JOIN.  This is the desired behaviour, so we can show transaction counts in months we haven't been billed for yet (or possibly even invoice counts when we've received no transactions in a month).  So I think the COALESCE is necessary.